plugins/inputs/logfile/tailersrc_test.go

```diff
@@ -213,56 +201,30 @@ func TestOffsetDoneCallBack(t *testing.T) {
 		if i == 10 { // Test before first truncate
 			time.Sleep(1 * time.Second)
 			b, err := ioutil.ReadFile(statefile.Name())
-			if err != nil {
-				t.Errorf("Failed to read state file: %v", err)
-			}
+			assert.NoError(t, err)
 			offset, err := strconv.Atoi(string(bytes.Split(b, []byte("\n"))[0]))
-			if err != nil {
-				t.Errorf("Failed to parse offset: %v, from '%s'", err, b)
-			}
-
-			if offset != 1010 {
-				t.Errorf("Wrong offset %v is written to state file, expecting 1010", offset)
-			}
+			assert.NoError(t, err)
+			assert.Equal(t, offset, 1010)
 		}
 
 		if i == 15 { // Test after first truncate, saved offset should decrease
 			time.Sleep(1 * time.Second)
-			log.Println(statefile.Name())
 			b, err := ioutil.ReadFile(statefile.Name())
-			log.Println(b)
-			if err != nil {
-				t.Errorf("Failed to read state file: %v", err)
-			}
+			assert.NoError(t, err)
 			file_parts := bytes.Split(b, []byte("\n"))
-			log.Println("file_parts: ", file_parts)
 			file_string := string(file_parts[0])
-			log.Println("file_string: ", file_string)
 			offset, err := strconv.Atoi(file_string)
-			log.Println(offset)
```

Comment:
Normally, I wouldn't want these log statements. But I do for tailersrc testing. What is the reason for removing these?

Reply:
No deep reason against it since I don't see any value of keeping it.

Reply:
Not entirely not relate. The `tailSrc` also uses the `tailFile`, so its kinda connivence for me to change this to.

Reply:
@adam-mateen would want your opinion on if you want these log statements in here or not.